Services/WebServices/ECS/classes/class.ilECSObjectSettings.php

```diff
@@ -119,31 +125,11 @@ protected function isActive(): bool
      */
     public function addSettingsToForm(ilPropertyFormGUI $a_form, $a_type): bool
     {
-        $this->logger->debug('Show ecs settings.');
-        if (!$this->isActive()) {
-            $this->logger->debug('Object type is not active. => no settings.');
-            return false;
-        }
+        $export_manager = ilECSExportManager::getInstance();
+        $exportable_participants = (new ilECSParticipantSettingsRepository())->getExportableParticipants($a_type);
 
-        $obj_id = $this->content_obj->getId();
```

Comment:
This variable is used in the rest of this method. If it is removed here, the rest of the usages need to be changed as well.